crates/sui-types/src/lib.rs

```diff
@@ -128,12 +134,20 @@ pub const SUI_RANDOMNESS_STATE_OBJECT_ID: ObjectID =
 pub const SUI_DENY_LIST_ADDRESS: AccountAddress = deny_list_addr();
 pub const SUI_DENY_LIST_OBJECT_ID: ObjectID = ObjectID::from_address(SUI_DENY_LIST_ADDRESS);
 
+/// 0x9: hardcode object ID for the singleton bridge object.
+pub const SUI_BRIDGE_ADDRESS: AccountAddress = address_from_single_byte(9);
```

Comment:
nit: This name is going to get quite confusing -- can we clarify that this is the bridge object? If someone asked me to tell you which is the package and which is the object out of `BRIDGE_ADDRESS` and `SUI_BRIDGE_ADDRESS` I would ...not be happy.